pkgs/by-name/po/polymc/package.nix

```diff
@@ -0,0 +1,115 @@
+{
+  addDriverRunpath,
+  flite,
+  gamemode,
+  glfw,
+  jdk17,
+  jdk21,
+  jdk8,
+  kdePackages,
+  lib,
+  libGL,
+  libX11,
+  libXcursor,
+  libXext,
+  libXrandr,
+  libXxf86vm,
+  libpulseaudio,
+  libusb1,
+  mangohud,
+  mesa-demos,
+  openal,
+  polymc-unwrapped,
+  stdenv,
+  symlinkJoin,
+  udev,
+  vulkan-loader,
+  wayland,
+  xrandr,
+
+  jdks ? [
+    jdk21
+    jdk17
+    jdk8
+  ],
+  enableTweaks ? stdenv.hostPlatform.isLinux,
```

Comment:
Is there any reason we're grouping gamemode and mangohud together here? I'd personally prefer having two separate options, and I don't see an issue with that, either.

Reply:
I personally don't see a problem with merging them together, since they can just be disabled in the GUI itself. Secondly, while separating them isn't terrible, if we were to extrapolate to dealing with 15 tweaks, adding 15 different options would be gratuitous. However, if you think this is the better approach, I'm fine with that.

Reply:
Well, I just don't think it makes much sense to mix the compile-time gamemode support option with the purely runtime mangohud support that only affects the wrapper, but sure.